### DISCUSSION TITLE
Llm.nvim support.

### DISCUSSION THREAD

**Discussion by kozer** [LazyVim/LazyVim#1497] [Q&A]
Does anyone have a working solution for llm.nvim plugin integrated with lazyvim?

**Comment by folke** [comment]
Just add it to your config?

**Comment by swaroopch** [comment]
Got it working, was keen to try this out myself 😄  - https://www.swaroopch.com/lazyvim-and-codellama/ (apologies if linking is not allowed)

**Comment by meicale** [comment]
Some local llm support is quite needed! Such as [llm.nvim](https://github.com/huggingface/llm.nvim) , [ChatGPT.nvim](https://github.com/jackMort/ChatGPT.nvim), [gp.nvim](https://github.com/Robitx/gp.nvim)  and [continue.dev](https://github.com/continuedev/continue/issues/917) which is going to support! Thank you!
